packages/eslint-plugin/src/configs/react.js

```diff
@@ -44,5 +44,11 @@ module.exports = {
         ],
       },
     },
+    {
+      files: '**/*.cy.{ts,tsx,js}',
+      rules: {
+        'import/no-extraneous-dependencies': 'off',
```

Comment:
A happy side effect of moving cypress test files from the `e2e/` folder to `src/` is that those files are actually being linted now. However, it now triggers an `import/no-extraneous-dependencies` linting error for some files which I decided to turn off for v9 packages since we probably don't want to be adding dependencies just for these cypress test files that aren't even being shipped. 

![image](https://user-images.githubusercontent.com/8649804/193088836-0b2cb304-1404-408e-8099-7d19f85b033f.png)